identity/webapp/src/frontend/components/Layout.style.ts

```diff
@@ -1,21 +1,28 @@
 import styled from 'styled-components';
+import Space from '@weco/common/views/components/styled/Space';
+import { grid } from '@weco/common/utils/classnames';
 
-export const Container = styled.div`
+export const Container = styled(Space).attrs({
+  v: { size: 'xl', properties: ['margin-bottom'] },
+})`
   background-color: white;
-  max-width: 782px;
-  margin: 0 auto;
-
-  @media screen and (min-width: 600px) {
-    margin: 64px auto;
-    border-radius: 10px;
-    padding-top: 1em;
-  }
+  border: 1px solid ${props => props.theme.color('pumice')};
+  border-radius: 10px;
 `;
 
 export const Wrapper = styled.div`
-  max-width: 32em;
-  margin: 0 auto;
   padding: 1em;
 `;
 
-export const Title = styled.h1.attrs({ className: 'font-wb font-size-2' })``;
+export const Header = styled(Space)`
+  background: ${props => props.theme.color('white')};
+`;
+
+export const Title = styled.h1.attrs({ className: 'font-wb font-size-0' })``;
```

Comment:
This is bigger than any other h1 we have. Do we definitely want it this big?

Reply:
It's the same size as the other header titles, have changed to use the font function though